radix-engine-tests/tests/preview.rs

```diff
@@ -27,25 +28,62 @@ fn test_transaction_preview_cost_estimate() {
         manifest,
         &preview_flags,
     );
+    let size_diff = manifest_encode(&notarized_transaction).unwrap().len()
+        - manifest_encode(&preview_intent.intent).unwrap().len();
 
     // Act & Assert: Execute the preview, followed by a normal execution.
     // Ensure that both succeed and that the preview result provides an accurate cost estimate
-    let preview_result = test_runner.preview(preview_intent, &network);
-    let preview_receipt = preview_result.unwrap();
-    preview_receipt.expect_commit_success();
-
-    let receipt = test_runner.execute_transaction(
+    let preview_receipt = test_runner.preview(preview_intent, &network).unwrap();
+    let preview_result = preview_receipt.expect_commit_success();
+    let actual_receipt = test_runner.execute_transaction(
         validate(&network, &notarized_transaction).get_executable(),
         FeeReserveConfig::default(),
-        ExecutionConfig::for_preview(),
+        ExecutionConfig::for_notarized_transaction()
```

Comment:
Can we explicitly have a `for_preview()` do this, and test with that configuration?

So we know we're using a mode which has been tested in the engine?

Reply:
We have `for_preview()` configuration.  

This test is about comparing the estimated results with **actual** results, which is why I picked `for_notarized_transaction()`.

Reply:
Oooh yes, good point!